### DISCUSSION TITLE
Full Site Editing Theme

### DISCUSSION THREAD

**Discussion by rimager** [timber/timber#2555] [General]
Apologies if this is the wrong place to ask. New to timber. Is there a theme that supports "Full Site Editing"/a block theme using timber that anyone is aware of that works well with 5.9? Thanks in advance.

**Comment by n3storm** [comment]
First, probably Timber is not for you if you ask that.
Second, you can easily implement a theme that support Full Site Editing then you will use like a 10% of Timber functionality.

Not trying to troll, just trying to save you time :)

**Comment by n3storm** [comment]
If I just answer "No" as you suggest I don't feel I am encouraging a Discussion or Debate as section name indicates.

The answer btw is "Maybe", have a look to https://flyntwp.com/ is based on Timber and Acf and supports full site editing via components.

Cheer up :)

**Comment by rmens** [comment]
I’m curious for this too. Made my own site editor with ACF and Timber, just like FlyntWP. But it would be interesting to see how others embrace the future of Wordpress with Timber.
